api/lib/redisHelper.mjs

```diff
@@ -60,7 +70,15 @@ export async function getStudent(email) {
             { expectedType: 'string', email },
         );
     }
-    return await getEntry(email);
+    let student;
+    try {
+        student = await getEntry(email);
+    } catch (error) {
+        throw new StudentNotFoundError(`Student ${email} is not in the database.`);
+    } finally {
+        //TODO: record metrics?
```

Comment:
Ooooo.  I like it!  Right now, I'm more concerned about usability metrics gathered from the API or proxy, but if you are curious to see how the db is being used, you're more than welcome to configure a metrics handler for that :D